src/util/migrations/20230710-practitioner-to-careteam.py

```diff
@@ -0,0 +1,75 @@
+import requests
+import json
+
+# TODO change or switch to parameter
+fhir_server = "https://fhir.isacc.dev.cirg.uw.edu/fhir"
```

Comment:
Minor change; should allow testing on another system (dev) before running on prod
```suggestion
import json
import requests
import sys

script_name = sys.argv.pop()

default_fhir_server = "https://fhir.isacc.dev.cirg.uw.edu/fhir"
# use first argument as FHIR server base URL, if passed
fhir_server = next(iter(sys.argv), default_fhir_server)
```